.github/workflows/mariadb.yml

```diff
@@ -0,0 +1,126 @@
+          path: mariadb
+          repository: MariaDB/server
+      - name: Remove bundled Mroonga
+        run: |
+          rm -rf mariadb/extra/groonga
+          rm -rf mariadb/storage/mroonga
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/extra/groonga
+          repository: groonga/groonga
+          submodules: recursive
+      - uses: actions/checkout@v4
+        with:
+          path: mariadb/storage/mroonga
+          repository: mroonga/mroonga
+          submodules: recursive
+      - name: Build MariaDB
+        # To reduce build time, we build only the MariaDB core and Mroonga.
+        # The following options disable all other storage engines, module-type
+        # plugins, and some core options to reduce compile time and external
+        # dependencies.
+        run: |
+          cmake \
+            -Smariadb \
+            -Bmariadb.build \
+            -G Ninja \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DCONC_WITH_UNITTEST=OFF \
+            -DMYSQL_MAINTAINER_MODE=OFF \
+            -DPLUGIN_ARCHIVE=NO \
+            -DPLUGIN_AUDIT_NULL=NO \
+            -DPLUGIN_AUTH_ED25519=NO \
+            -DPLUGIN_AUTH_TEST_PLUGIN=NO \
+            -DPLUGIN_AUTH_0X0100=NO \
+            -DPLUGIN_BLACKHOLE=NO \
+            -DPLUGIN_COLUMNSTORE=NO \
+            -DPLUGIN_CONNECT=NO \
+            -DPLUGIN_DAEMON_EXAMPLE=NO \
+            -DPLUGIN_DEBUG_KEY_MANAGEMENT=NO \
+            -DPLUGIN_DIALOG_EXAMPLES=NO \
+            -DPLUGIN_DISKS=NO \
+            -DPLUGIN_EXAMPLE=NO \
+            -DPLUGIN_EXAMPLE_KEY_MANAGEMENT=NO \
+            -DPLUGIN_FEDERATED=NO \
+            -DPLUGIN_FEDERATEDX=NO \
+            -DPLUGIN_FILE_KEY_MANAGEMENT=NO \
+            -DPLUGIN_FTEXAMPLE=NO \
+            -DPLUGIN_FUNC_TEST=NO \
+            -DPLUGIN_HANDLERSOCKET=NO \
+            -DPLUGIN_INNOBASE=NO \
+            -DPLUGIN_LOCALES=NO \
+            -DPLUGIN_METADATA_LOCK_INFO=NO \
+            -DPLUGIN_OQGRAPH=NO \
+            -DPLUGIN_PASSWORD_REUSE_CHECK=NO \
+            -DPLUGIN_PERFSCHEMA=NO \
+            -DPLUGIN_QA_AUTH_INTERFACE=NO \
+            -DPLUGIN_QA_AUTH_SERVER=NO \
+            -DPLUGIN_QA_AUTH_CLIENT=NO \
+            -DPLUGIN_QUERY_CACHE_INFO=NO \
+            -DPLUGIN_QUERY_RESPONSE_TIME=NO \
+            -DPLUGIN_ROCKSDB=NO \
+            -DPLUGIN_SEQUENCE=NO \
+            -DPLUGIN_SERVER_AUDIT=NO \
+            -DPLUGIN_SIMPLE_PASSWORD_CHECK=NO \
+            -DPLUGIN_SPHINX=NO \
+            -DPLUGIN_SPIDER=NO \
+            -DPLUGIN_SQL_ERRLOG=NO \
+            -DPLUGIN_TEST_SQL_DISCOVERY=NO \
+            -DPLUGIN_TEST_SQL_SERVICE=NO \
+            -DPLUGIN_TEST_VERSIONING=NO \
+            -DPLUGIN_TYPE_TEST=NO \
+            -DPROVIDER_LZ4=NO \
+            -DWITH_EMBEDDED_SERVER=NO \
+            -DWITH_UNIT_TESTS=OFF \
+            -DWITH_MARIABACKUP=OFF \
+            -DWITH_SAFEMALLOC=OFF \
+            -DWITH_WSREP=OFF \
+            -DWITHOUT_DYNAMIC_PLUGIN=ON
```

Comment:
Does this mean that we don't build `ha_mroonga.so`?
We must test `ha_mroonga.so`.

Reply:
fix: [775dfad](https://github.com/mroonga/mroonga/pull/896/commits/775dfadc7926a90684b9025e4e3831c30974eea8) 

When I specified this option. I could build `ha_mroonga.so` as follows.
But I think this isn't expected behavior because Mroonga is included in dynamic plugin too using `cmake -L`.
So I've removed this option here.
```console
$ find ./mariadb-11.4.5-debug.build | grep "ha_mroonga.so"
./mariadb-11.4.5-debug.build/storage/mroonga/ha_mroonga.so
$ /usr/local/mysql/bin/mariadb < /usr/local/mysql/share/mroonga/install.sql
$ /usr/local/mysql/bin/mariadb -e "SHOW ENGINES" | grep Mroonga
Mroonga	YES	CJK-ready fulltext search, column store	NO	NO	NO
```

```console
$ cmake -L . | grep PLUGIN_MROONGA
PLUGIN_MROONGA:STRING=DYNAMIC
```

Reply:
I think I couldn't understand this part well now.

https://github.com/MariaDB/server/blob/22efc2c784e1b7199fb5804e6330168277ea7dce/storage/mroonga/CMakeLists.txt#L53-L64